src/main/java/com/northconcepts/datapipeline/examples/cookbook/StreamArrayValues.java

```diff
@@ -0,0 +1,35 @@
+package com.northconcepts.datapipeline.examples.cookbook;
+
+import com.northconcepts.datapipeline.core.ArrayValue;
+import com.northconcepts.datapipeline.core.Record;
+
+public class StreamArrayValues {
+
+    public static void main(String[] args) {
+        Record record = new Record();
+
+        record.setField("stageName", "John Wayne");
+        record.setField("realName", "Marion Robert Morrison");
+        record.setField("gender", "male");
+        record.setField("city", "Winterset");
+        record.setField("balance", 156.35);
+
+        ArrayValue arrayValue = new ArrayValue();
+        record.forEach(field -> arrayValue.addValue(field));
```

Comment:
This is an unusual way of creating an ArrayValue.  Since most users will be accessing an array from a record, let's create the example around that.  Consider something like: `record.setField("balance", new double[]{156.35, 123.45});`.

And then use `Record.getFieldValueAsArray("balance", null)`.